numpy-stubs/core/numeric.pyi

```diff
@@ -1,111 +0,0 @@
-from types import ModuleType
-from typing import (
-    Sequence,
-    Optional,
-    TypeVar,
-    Union,
-    Tuple,
-    List,
-    Iterable,
-    Callable,
-    Any,
-)
-
-from numpy import ndarray, dtype, _ArrayLike, _ShapeLike
-
-_T = TypeVar("_T")
-
-def zeros_like(
-    a: _ArrayLike,
-    dtype: Optional[dtype] = ...,
-    order: str = ...,
-    subok: bool = ...,
-    shape: Optional[Union[int, Sequence[int]]] = ...,
-) -> ndarray[int]: ...
```

Comment:
Converted the various `ndarray[T]` things to `ndarray`, since that's not a thing yet (https://github.com/numpy/numpy-stubs/pull/48).